redfish/config.py

```diff
@@ -16,7 +16,22 @@
 
 logger = None
 TORTILLADEBUG = True
-REDFISH_LOGFILE = "/var/log/python-redfish/python-redfish.log"
+HOME = os.getenv('HOME')
+if HOME == '':
+    print("$HOME environment variable not set, please check your system")
+    sys.exit(1)
+
+REDFISH_HOME = HOME + "/.redfish"
+if not os.path.exists(REDFISH_HOME):
+    try:
+        os.mkdir(REDFISH_HOME)
+    except IOError:
+        print('ERROR: can\'t create {}.\n'.format(REDFISH_HOME))
+        print('       Try to create directory {}'.format(os.path.dirname(REDFISH_LOGFILE)))
+        print('       using: mkdir -p {}'.format(os.path.dirname(REDFISH_LOGFILE)))
+        sys.exit(1)
+
+REDFISH_LOGFILE = REDFISH_HOME + "/python-redfish.log"
```

Comment:
If it's a good practice, it then wasn't applied here as I copied what existed anda adapted ;-)